code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -3759,3 +3759,78 @@ datum
 			alch_strength = 0.4
 			description = "A warm, late night drink, usually enjoyed during long winter nights."
 			reagent_state = LIQUID
+
+		fooddrink/grenadine
+			name = "Grenadine"
+			id = "grenadine"
+			fluid_r = 234
+			fluid_g = 19
+			fluid_b = 19
+			description = "A sticky, sweet and tart non-alcoholic bar syrup, used in cocktails for it's distinct bright red colour."
+			reagent_state = LIQUID
+
+		fooddrink/lemonade/pinklemonade
```

Comment:
I would typically call this `fooddrink/lemonade/pink`, but it's hardly critical so NBD.